packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -34,13 +35,17 @@ const AppointmentActions: React.FC<AppointmentActionsProps> = ({ appointment })
   );
 
   const handleCheckout = () => {
-    const dispose = showModal('end-visit-dialog', {
-      closeModal: () => {
-        mutateVisit();
-        dispose();
-      },
-      patientUuid,
-    });
+    if (checkOutButton.customUrl) {
+      navigate({ to: checkOutButton.customUrl, templateParams: { patientUuid, appointmentUuid: appointment.uuid } });
```

Comment:
Be explicit in the config description (in config-schema.ts) that the the customUrl supports those 2 template params.